test/suites/drift/suite_test.go

```diff
@@ -849,7 +852,17 @@ var _ = Describe("Drift", func() {
 			Eventually(func(g Gomega) {
```

Comment:
Do we even care that the NodeClaim count is the same or do we just care that the tainted node is deleted? I think we just want to make sure that we get through the loop and that we still maintain the same set of nodes after the full disruption loop occurs. Realistically, I don't even know if we really need to validate that we remove the old node. Ideally, we could run a check throughout the whole test to validate that we never delete our nodes and we maintain static stability rather than just checking at the end.

Reply:
That's a good point, it is valuable to test this IMO but that can be left to a registration TTL test. Plus removing this wait should trim ~4 minutes off of the drift suite.

Reply:
I do wonder if there's value in checking the nodes consistently throughout the test vs just testing that they still exist in the end. It definitely doesn't hurt, but introducing a goroutine to poll cluster state throughout the test is more complicated than what we have today. Do we expect there to be a scenario where we delete a node and provision a replacement with the same name?

Reply:
Yeah, that's a good solid point. Since we are checking on UUIDs, there's really no difference.